apache/superset
# feat(helm): Make Chart.AppVersion default value for image tag

### SUMMARY
The PR adds `appVersion` to `Chart.yaml`, pins default Superset version to `2.0.1` and uses it as a default value for image tag. The same logic is used in many other official helm charts.
This change also eliminates confusing `helm search repo` output saying chart deploys Superset 1.0:
```
$ helm search repo
NAME              CHART VERSION   APP VERSION     DESCRIPTION
argo/argo-cd        5.19.6          v2.5.7          A Helm chart for Argo CD, a declarative, GitOps...
...
minio/operator      4.5.8           v4.5.8          A Helm chart for MinIO Operator
...
superset/superset   0.8.4           1.0             Apache Superset is a modern, enterprise-ready b...
```

### TESTING INSTRUCTIONS
Check that the right value is used by default `helm template test ./helm/superset`
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: test-superset
  ...
spec:
  containers:
    - name: superset
      image: "apache/superset:2.0.1"
```
and with custom image tag `helm template test ./helm/superset --set image.tag=custom`
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: test-superset
  ...
spec:
  containers:
    - name: superset
      image: "apache/superset:custom"
```

### ADDITIONAL INFORMATION
